scripts/build_imagemagick

```diff
@@ -13,56 +13,23 @@ function handle_error() {
 }
 trap handle_error EXIT SIGINT
 
-function add_xenial_repositories() {
```

Comment:
Even when building for Heroku-20, this script was reaching back to `xenial` so it could `apt-get install libjasper-dev` for imagemagick. That library has been superseded by https://github.com/uclouvain/openjpeg which is distributed with Ubuntu 24.04

```bash
$ ldconfig -p | grep libopenjp2
libopenjp2.so.7 (libc6,x86-64) => /lib/x86_64-linux-gnu/libopenjp2.so.7
libopenjp2.so (libc6,x86-64) => /lib/x86_64-linux-gnu/libopenjp2.so
```